src/applications/mhv/secure-messaging/tests/e2e/secure-messaging-navigate-away-from-replying-message.cypress.spec.js

```diff
@@ -2,7 +2,8 @@ import SecureMessagingSite from './sm_site/SecureMessagingSite';
 import PatientInboxPage from './pages/PatientInboxPage';
 import { AXE_CONTEXT, Locators } from './utils/constants';
 
-describe('verify signature', () => {
+// test have to be refactored after according to new scenario (modal warning should appear)
+describe.skip('verify signature', () => {
```

Comment:
@vsaleem please see if @fazilqa may assist with refactoring in this same PR. Otherwise, I can try to assist

Reply:
@oleksii-morgun-va, @fazilqa plans to update this test and refactor the other existing tests once these changes make it in to staging.  QA has also completed testing on this as of 10/5.